### ISSUE TITLE
Components code reorganisation / cleanup

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/web-components#210]
# Discussion 🗣
All components have the following general structure, where several things seem to be unused / dead code:
```
.eslintrc
.gitignore
.lintstagedrc
.nvmrc
.stylelintrc
README.md
babel.config.rollup.js
jsconfig.json
netlify.toml // UNUSED
package-lock.json
package.json
prepare.js // UNUSED
public // UNUSED
rollup.config.js
src
  index.js // UNUSED
  lib
    index.js
    index.test.js
    ... // REST OF THE CODE
  setupTests.js // UNUSED?
test
```

Is there a reason for this code organization? Could we perhaps consolidate into something more like:
```
.eslintrc
.gitignore
.lintstagedrc
.nvmrc
.stylelintrc
README.md
babel.config.rollup.js
jsconfig.json
package-lock.json
package.json
rollup.config.js
src
  index.js
  index.test.js
  ... // REST OF THE CODE
test
```

**Comment by jeromesimeon** [comment]
@irmerk @DianaLease Would love your insight / input / thoughts on this.

**Comment by irmerk** [comment]
I think this is a good idea, however a couple points I'd like @DianaLease's input on:

```sh
.eslintrc
.gitignore
.lintstagedrc
.nvmrc
.stylelintrc
README.md
babel.config.rollup.js
jsconfig.json
netlify.toml // UNUSED and can be removed
package-lock.json
package.json
prepare.js // UNUSED and can be removed
public // UNUSED and I think can be removed
rollup.config.js
src
  index.js // I believe this is used
  lib
    index.js
    index.test.js
    ... // REST OF THE CODE
  setupTests.js // UNUSED and I think can be removed
test
```

**Comment by jeromesimeon** [comment]
> I think this is a good idea, however a couple points I'd like @DianaLease's input on:
> 
> ```shell
> src
>   index.js // I believe this is used
> ```

Pretty sure it's not? Where is it used?

**Comment by jeromesimeon** [comment]
Approved / fixed in #209
